syncany-lib/src/main/java/org/syncany/operations/down/actions/ChangeFileSystemAction.java

```diff
@@ -28,78 +29,86 @@
 	public ChangeFileSystemAction(Config config, FileVersion fromFileVersion, FileVersion toFileVersion, MemoryDatabase winningDatabase) {
 		super(config, winningDatabase, fromFileVersion, toFileVersion);
 	}
-	
+
 	@Override
-	public FileSystemActionResult execute() throws Exception {
+	public FileSystemActionResult execute() throws IOException {
 		boolean fromFileExists = fileExists(fileVersion1);
 		boolean fromFileMatches = fromFileExists && fileAsExpected(fileVersion1);
-		
+
 		boolean toFileExists = fileExists(fileVersion2);
 		boolean toFileMatches = toFileExists && fileAsExpected(fileVersion2);
-		
+
 		boolean filesAtSameLocation = fileVersion1.getPath().equals(fileVersion2.getPath());
 
 		if (fromFileMatches && !toFileMatches) { // Normal case	
 			// Original file matches, so we can delete it
 			// Create conflict file for winning file, if it exists
 			if (!toFileExists) {
-				logger.log(Level.INFO, "     - (1) Original file matches, target file does NOT match: deleting original file, creating target file at: "+fileVersion2);
-				
-				deleteFile(fileVersion1);	
-				createFileFolderOrSymlink(fileVersion2);						
+				logger.log(Level.INFO,
+						"     - (1) Original file matches, target file does NOT match: deleting original file, creating target file at: "
```

Comment:
Jup, so saving it causes this to be messed up again (I have format on save enabled). Would you like me to edit this in vim or fix it in some other way?